__tests__/app.test.js

```diff
@@ -32,4 +32,26 @@ describe('api', () => {
         })
       })
   })
+  test('200: GET request responds with array of review objects', () => {
+    return request(app)
+      .get('/api/reviews')
+      .expect(200)
+      .then(({body:{reviews}}) => {
+        expect(reviews).toBeInstanceOf(Array)
```

Comment:
Can leave this in but you're checking that this is an Array then immediately checking the length and then using an array method - implying that this is an array anyway. Something to bear in mind for future :)